home.html

```diff
@@ -41,31 +42,138 @@ <h1>
+                am able to build any kind of website using these technologies.
+              </p>
+            </div>
+          </div>
+
+          <div class="expitem">
+            <div class="inneritem">
+              <img
+                src="assets/work-backend.png"
+                alt=""
+                width="60"
+                height="60"
+              />
+            </div>
+            <div class="inneritem">
+              <h3>Back end</h3>
+            </div>
+            <div class="inneritem">
+              <p>
+                I have experience in Node js and PHP for building backend
+                projects, therefore I am able to build any kind of back project.
+              </p>
+            </div>
+          </div>
+          <div class="expitem">
+            <div class="inneritem">
+              <img
+                src="assets/work-crossplatform.png"
+                alt=""
+                width="60"
+                height="60"
+              />
+            </div>
+            <div class="inneritem">
+              <h3>Cross platform mobile</h3>
+            </div>
+            <div class="inneritem">
+              <p>
+                I have experience in Flutter , there for I am able to build any
+                kind of mobile application using flutter.
+              </p>
+            </div>
+          </div>
+        </section>
+        <!-- End work experience -->
+        <!-- Langages and frameworks -->
+
+        <!-- End languages and frameworks -->
+        <section class="lan-fra">
+          <h5>Languages and frameworks</h5>
+          <ul class="list">
+            <li class="listitem">
+              <div class="imgcontainer">
+                <img
+                  src="assets/javascriptlogo.png"
+                  alt=""
+                  width="35"
+                  height="35"
+                />
+              </div>
+              <div class="txtcontainer">
+                <p>Java script</p>
+              </div>
             </li>
-            <li>
-              <a href="#"
-                ><img src="assets/github-brands.svg" alt="" class="icons"
-              /></a>
+            <li class="listitem">
+              <div class="imgcontainer">
+                <img src="assets/html5logo.png" alt="" width="35" height="35" />
+              </div>
+              <div class="txtcontainer">
+                <p>HTML 5</p>
+              </div>
             </li>
-            <li>
-              <a href="#"
-                ><img src="assets/angellist-brands.svg" alt="" class="icons"
-              /></a>
+            <li class="listitem">
+              <div class="imgcontainer">
+                <img src="assets/csslogo.png" alt="" width="35" height="35" />
+              </div>
+              <div class="txtcontainer">
+                <p>Css</p>
+              </div>
             </li>
           </ul>
         </section>
```

Comment:
- [ ] This section also looks different from the original design. 
![image](https://user-images.githubusercontent.com/43698511/129272465-ada9e3af-fa4c-49f0-b6f6-b1539016587c.png)

![image](https://user-images.githubusercontent.com/43698511/129273369-665d6568-28cc-4a6b-a477-7e530f258719.png)

As you can see in the pictures. Your section is missing

- [ ]  The About Myself title
- [ ] The presentation paragraph
- [ ] The let's connect link
- [ ] the list of clickable media icons
- [ ] The Get my resume button
- [ ] The titles Languages, Frameworks, and skills. Kindly add all the missing elements.
- [ ] Kindly respect the order of the sections.